Dockerfile

```diff
@@ -9,14 +9,11 @@ RUN apt-get update && apt-get dist-upgrade -y && \
 
```

Comment:
If you also fix the Dockerfile to use apt-get instead of apt for python, I could forget my PR #6